Kconfig

```diff
@@ -216,3 +216,12 @@ menu "Test suite"
 depends on FEATURE_RUNNABLE_PROGRAMS
 source "src/test/Kconfig"
 endmenu
+
+menu "Bindings"
+config NODE_BINDINGS
```

Comment:
you better call it `USE_NODEJS`, it's more in the lines of what we did. Also calling it `NODEJS` instead of just `NODE` in Makefile and Kconfig may help others to understand what it means (Soletta's Flow already use the term 'node').